utils/frame/remote-externalities/src/lib.rs

```diff
@@ -161,7 +175,13 @@ pub struct OnlineConfig {
 impl Default for OnlineConfig {
 	fn default() -> Self {
 		Self {
-			uri: "http://localhost:9933".into(),
+			uri: TARGET.to_owned(),
+			rpc: Arc::new(
+				HttpClient::new(
+					TARGET,
```

Comment:
okay I approved but this is very wrong :D you must usethe given uri, not the hardcoded stuff. I will have to fix this a bit.